### DISCUSSION TITLE
`GRID_CELL_CROP_FRACTION`

### DISCUSSION THREAD

**Discussion by unhandyandy** [iansan5653/open-mcr#83] [Q&A] [answered]
The comments say when this is 0, the circle touches the cell sides,
and when this is 0.5, the circle reaches halfway from the center to the sides.
The code sets
GRID_CELL_CROP_FRACTION = 0.25
Does that mean the circle goes 75% of the way from the center to the sides?

**Comment by iansan5653** [comment, marked as answer]
Hello, sorry for the very late reply - I completely missed this notification. That sounds right though - the circle diameter should be 75% of the grid cell side length.

**Comment by unhandyandy** [comment]
Thanks.  But you realize that a lot of people will think it would only go 25%.

**Reply by iansan5653** [reply]
I think that's a good point - would you like to submit a pull request that changes the implementation?

**Reply by unhandyandy** [reply]
I'm not at the point of writing code for this project yet.
Btw, you could also just change the name of the variable.

**Reply by iansan5653** [reply]
What do you think would be a better name?
